### ISSUE TITLE
[Bug] When I click another open program on the task bar it opens then minimizes (closing LES fixes the issue)

### ISSUE THREAD

**Issue by gsirvine** [LiveEnhancementSuite/LESforWindows#6]
**Describe the bug**
I get this problem where, when I click anything on the task bar for example another program while LES is running and it minimizes that program real fast.  This is very  cumbersome.  I wasn't sure if this was meant to be some sort of productivity feature that is meant to keep focus on Ableton but i don't have focus issues like that and sometimes I use other music production software simultaneously. 

So if I am running Ableton and iZotope RX and switch to RX via mouse clicking it on the taskbar it minimizes it real fast as soon as it pops up.  Problem goes away if I close LES.  If I move my mouse off the task bar real fast after clicking something this also is a workaround.

**Desktop (please complete the following information):**
 - OS: Windows 8
 - Version 8.0

**Comment by Inversil** [comment]
Hmm this is strange; I can't reproduce this even with izotope RX. It might be a bug with something in windows 8 interfering with the script. I'm not doing anything special when you deselect ableton. 

Does pressing middle click or alt or control together in any combination cause the window to minimize?

**Comment by gsirvine** [comment]
Thanks for your response.  I tried what you said and it didn't work.

But I did find there is an interaction between the software Actual Window Manager and LES.  I enabled settings within Actual window manager one at a time and discovered that disabling the Actual Window Manager Taskbar replacement is the only fix for this.  Everything else in Actual Window Manager seems to work fine and not interact negatively with LES.  Issue only happens between these two softwares (AWM and LES) and nothing else.  Who knows.  Also there is a newer version of AWM that I don't own and maybe the problem is on their end and already squashed.  Definitely probably rare for people to use both of these softwares.    

Either way I found a workaround.  Maybe my discover will help someone else running both of these softwares that is ultra confused by this behavior.

For reference disable the multi-monitor task bar in AWM and find a way to work with the native windows one.
